template.yaml

```diff
@@ -28,7 +28,13 @@ Globals:
     Runtime: nodejs16.x
 
 Resources:
-  HelloWorld:
+  ZendeskWebhook:
     Type: AWS::Serverless::Function
     Properties:
-      Handler: helloWorld.handler
+      Handler: zendeskWebhook.handler
```

Comment:
Same as the comment above, could this just be `validateRequest.handler` if we think that's a good name.